.devcontainer.json

```diff
@@ -0,0 +1,7 @@
+// See https://aka.ms/vscode-remote/devcontainer.json for format details.
+{
```

Comment:
This file is for use with the [VS Code Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension. I was using it to run some tests in the Docker container locally.

I'll probably delete it before this is merged.